index.js

```diff
@@ -29,7 +30,7 @@ var isLiteral = require('nlcst-is-literal');
  *   break the middleware.
  */
 function all(tree, file, config, next) {
-    var hun = config.dictionary;
+    spellchecker.use(config.dictionary);
     var ignoreLiteral = config.ignore;
     var queue = 0;
```

Comment:
Ah, that makes sense. I was trying to reason about what those were doing, and it wasn't adding up. :-P